AEPCampaignClassic/Tests/UnitTests/CampaignClassicPublicAPITests.swift

```diff
@@ -40,24 +43,24 @@ class CampaignClassicPublicAPITests: XCTestCase {
             let isRegisterDeviceEvent = try? XCTUnwrap(event.data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.REGISTER_DEVICE] as? Bool)
             let deviceToken = try? XCTUnwrap(event.data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_TOKEN] as? String)
             let userKey = try? XCTUnwrap(event.data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.USER_KEY] as? String)
-            let additionalParameters = try? XCTUnwrap(event.data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.ADDITIONAL_PARAMETERS] as? [String : String])
+            let additionalParameters = try? XCTUnwrap(event.data?[CampaignClassicConstants.EventDataKeys.CampaignClassic.ADDITIONAL_PARAMETERS] as? [String : AnyCodable])
             
             // verify
             XCTAssertEqual(isRegisterDeviceEvent, true)
             XCTAssertEqual(deviceToken, self.SAMPLE_PUSHTOKEN_AS_HEXSTRING)
             XCTAssertEqual(userKey, SAMPLE_USER_KEY)
-            XCTAssertEqual(additionalParameters, self.SAMPLE_ADDITIONAL_DATA)
+            XCTAssertEqual(additionalParameters, ANYCODABLE_ADDITIONAL_DATA)
```

Comment:
XCAssert comparison of [String: Any] Dictionary is not possible today, because `Any` is not equatable..
I think its easier to keep the same way 
And Equatable on AnyCodable should have been tested thoroughly on Core

Reply:
nit: You could 1) compare their sizes and do a pair-wise element comparison (probably not ideal) 2) Use your own test util to convert to NSDictionary for comparison, see https://github.com/adobe/aepsdk-optimize-ios/blob/dev/Tests/AEPOptimizeTests/TestHelpers/Dictionary%2BOptimizeTests.swift. I'll leave it up to you.